frontend/src/metabase/visualizations/lib/apply_tooltips.js

```diff
@@ -95,7 +95,9 @@ export function getClickHoverObject(
     // Loop over *all* of the columns and create the new array
     if (rawRow) {
       data = rawCols.map((col, i) => {
-        if (isNormalized && cols[1].field_ref === col.field_ref) {
+        const isSeriesValueColumn = cols[1].field_ref === col.field_ref;
```

Comment:
@paulrosenzweig is it possible to have `field_ref` null or undefined?

Reply:
I can't think of a situation where that would be undefined. Even if it's a native query or aggregated value in a GUI query, it should have a field ref.

This isn't code you wrote, but comparing objects with `===` makes me nervous. I guess we're fine here if it's been working, but it might be worth the perf hit to use `_.isEquals` instead.

Reply:
> This isn't code you wrote

It turns out that I wrote it. 😆 Though, prior to my change it was comparing columns with `===`.

Reply:
@paulrosenzweig thanks for the answer! I also changed the comparison to `_.isEquals` as you suggested